src/pages/ZoukPersonaQuizPage.tsx

```
@@ -0,0 +1,313 @@
+                  <p className="text-xl text-primary font-medium mb-6">{result.subtitle}</p>
+
+                  <p className="text-gray-300 text-lg leading-relaxed mb-10 max-w-lg mx-auto border-t border-b border-white/10 py-6">
+                    {result.description}
+                  </p>
+
+                  <div className="flex flex-col sm:flex-row gap-4 justify-center">
+                    <button
+                      onClick={shareResult}
+                      className="btn bg-white text-black hover:bg-gray-200 flex items-center justify-center gap-2 px-8 py-3 rounded-full font-bold transition-all"
+                    >
+                      <Share2 size={20} />
```

Biome (2.3.13)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)